Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor/App.xaml.cpp

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+
+#include "App.xaml.h"
+#include "MainWindow.xaml.h"
+
+using namespace winrt;
```

Comment:
For the .xaml.cpp files, the recommendation is to define the `using namespace` statments inside a `namespace winrt { }`: See the [template](https://github.com/microsoft/WindowsAppSDK-Samples/blob/67cb2ca49dc1d07bcbbc6d4f869687385b0a8d73/Templates/VSIX/Project%20Templates/cpp-winui-template/App.xaml.cpp#L9) files as an example

Reply:
Fixed